src/sentry/dynamic_sampling/tasks.py

```diff
@@ -0,0 +1,82 @@
+import logging
+from typing import Sequence
+
+from sentry import features, quotas
+from sentry.dynamic_sampling.models.adjustment_models import AdjustedModel
+from sentry.dynamic_sampling.models.adjustment_models import DSProject as DSProject
+from sentry.dynamic_sampling.prioritise_projects import fetch_projects_with_total_volumes
+from sentry.dynamic_sampling.rules.helpers.prioritise_project import _generate_cache_key
+from sentry.dynamic_sampling.rules.utils import get_redis_client_for_ds
+from sentry.models import Organization, Project
+from sentry.tasks.base import instrumented_task
+from sentry.tasks.relay import schedule_invalidate_project_config
+from sentry.utils import metrics
+
+CHUNK_SIZE = 1000
+MAX_SECONDS = 60
+
+logger = logging.getLogger(__name__)
+
+
+@instrumented_task(
+    name="sentry.dynamic_sampling.tasks.prioritise_projects",
```

Comment:
I would add timeouts to all of those tasks unless you are provisioning dedicated celery workers (not just queues) for this new feature

Reply:
@untitaker it's not clear do I need seperatly deploy a new queue name?

Reply:
I believe you need to separately deploy a new queue name and ask ops to create a separate pair of workers. but just go `ops ticket me` and they will tell you exactly

Reply:
For simplicity, let's please go with a timeout. That will also alert us in Sentry if the task is taking longer than we would expect it to. The task itself does not consume a lot of compute, so we can keep it in existing queues for now.